src/modules/assertions/require_inline_coding.ts

```diff
@@ -0,0 +1,17 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+export const  requireInlineCoding = () => {
+  
+  const solfileRegex = /http:\/\/\S+.sol/g;
+  const prBodyString = (context.payload?.pull_request?.body != null) ? JSON.stringify(context.payload?.pull_request?.body) : null ;
+  const prBodyRegexMatches = (prBodyString != null ) ? prBodyString.match(solfileRegex) : null ; 
```

Comment:
This is equivalent.
```suggestion
  const prBodyRegexMatches = prBodyString?.match(solfileRegex);
```

Reply:
You marked this resolved but without comment or change.  I'm going to assume it was an accident and unresolve this.  If you intentionally marked it as resolved without change, can you please provide information why?

Reply:
Sorry, I meant not to mark it. But this code was fixed the day after, since then I have been trying to test it, no success due to not being able to get the eip-bot responding. If we manage to test it, this will help to close at least 5 more issues. Auto merge bot is ignoring me, however I managed to get answer from add-auto-merge-label and bot greeting (in JEAlfonsoP/EIPs/.github/workflows).
But, now I presume there will not be more need for update/coding to the EIP- bot since Sam Wilson is coding another bot ? (question)

Reply:
We hadn't heard from you on this issue for a while so Sam may have since implemented some of these checks already:

It is weird man, I was present in every call (ECH and EIPIP) for the last 6 weeks (easy to check with the recordings), always mentioning during my brief air time: I was trying to test the PRs code and setting the testing environment with nothing or even very small progress. Additional I was annoying Panda and Alita (and You) for setting a proper testing env, which allow me to progress in a very basic way.
But, thats Ok, I will work with him and see what can I provide for help and support to him. All that I can support I will.